salesforce-oms/1.1/modules/ROOT/pages/sap-s4hana-setup-guide.adoc

```diff
@@ -0,0 +1,229 @@
+| BusinessPartner | YY1_SME_MPARTYROLEID_bus | Text | 36 | Y | Y
+| SalesOrder | YY1_SME_MORDERID_SDH | Text | 36 | Y | Y
+| Product | YY1_SME_UnvProductId_PRD | Text | 36 | Y | Y
+|===
+
+=== Configure SAP S/4HANA Order Management
+
+Follow the steps in these sections to configure your Salesforce OMS to ERP Integration.
+
+==== Configure the Communication Arrangement
+
+Use the settings in the table to configure the communication arrangements.
+
+[%header%autowidth.spread]
+|===
+| Communication Arrangement ID | Service Name | Protocol 
+| SAP_COM_0109 | Sales Order (A2X) | OData V2
+| SAP_COM_0223 | Outbound Delivery - Notify (B2B) | OData V2
+| SAP_COM_0905 | Transportation Management - Freight Order Integration | OData V4
+| SAP_COM_0120 | Billing Document - Create with SD Document Reference | SOAP
+| SAP_COM_0120 | Billing Document - Read, Cancel, GetPDF| OData V2
+|===
+
+==== Add Required Entry for Outbound Delivery - Notify (B2B) Communication Arrangement
+
+Use the Fiori app Output Parameter Determination to add the following entry under the rule for *Outbound delivery* and *Determination Step Output Relevance*.
+
+[%header%autowidth.spread]
+|===
+| Output Type | Goods movement status | Document Category | Picking status 
+| DELIVERY_NOTE | A | J | C
+|===
+
+After configuring the communication arrangements, add the customer by using the rule `Outbound Delivery` and the Determination Step Channel.
+
+[%header%autowidth.spread]
+|===
+| Output Type | Recipient | Channel | Rolee 
+| DELIVERY_NOTE | <Customer Number> | EDI | WE
+|===
+
+Next, add the customer in the *Business Partner* section of the communication system, which is used to configure this communication arrangement.
+
+==== Custom CDS View
+
+* *Name*
++
+YY1_SDDocumentFlow
+
+* *Representative Key*
++
+SalesDocument
+
+* *Scenario*
++
+External API
+
+* *Data Sources*
++
+[%header%autowidth.spread]
+|===
+| Field name | Type | Cardinality | Join Condition 
+| I_SalesDocument| Primary Datasource | N/A | N/A
+| I_SDDocumentMultiLevelProcFlow| Associated Datasource | Zero or More [0..*] | PrecedingDocument = SalesDocument
+| I_SDDocumentCategory| Associated Datasource | Zero or More [0..*]| SDDocumentCategory = SubsequentDocumentCategory
+|===
+
+* *Elements*
++
+[%header%autowidth.spread]
+|===
+| Alias | Path | Type | Key 
+| SalesDocument| I_SalesDocument.SalesDocument| CHAR (10) | Yes
+| PrecedingDocumentCategory | _I_SDDocumentMultiLevelProcFlo.PrecedingDocumentCategory | CHAR (4) | No
+| SubsequentDocument | _I_SDDocumentMultiLevelProcFlo.SubsequentDocument | CHAR (10)| No
+| SubsequentDocumentCategory | _I_SDDocumentMultiLevelProcFlo.SubsequentDocumentCategory | CHAR (10)| No
+| ProcessFlowLevel | _I_SDDocumentMultiLevelProcFlo.ProcessFlowLevel | NUMC(2) | No
+| CreationDate | _I_SDDocumentMultiLevelProcFlo.CreationDate | DATS (8) | No
+| CreationTime | _I_SDDocumentMultiLevelProcFlo.CreationTime | TIMS (6) | No
+|===
+
+==== Create Custom Communication Scenario
+
```

Comment:
Should this be Create a Custom Communication Scenario? I read it as if "Create Custom Communication" was the name of a scenario the first time I read it. :)